easybuild/easyconfigs/f/FSL/FSL-6.0.2_probtrackx2_gpu_large_mri_support.patch

```diff
@@ -0,0 +1,3883 @@
+-    ("the University").
+-
+-    The Software is distributed "AS IS" under this Licence solely for
+-    non-commercial use in the hope that it will be useful, but in order
+-    that the University as a charitable foundation protects its assets for
+-    the benefit of its educational and research purposes, the University
+-    makes clear that no condition is made or to be implied, nor is any
+-    warranty given or to be implied, as to the accuracy of the Software,
+-    or that it will be suitable for any particular purpose or for use
+-    under any specific conditions. Furthermore, the University disclaims
+-    all responsibility for the use which is made of the Software. It
+-    further disclaims any liability for the outcomes arising from using
+-    the Software.
+-
+-    The Licensee agrees to indemnify the University and hold the
+-    University harmless from and against any and all claims, damages and
+-    liabilities asserted by third parties (including claims for
+-    negligence) which arise directly or indirectly from the use of the
+-    Software or the sale of any products based on the Software.
+-
+-    No part of the Software may be reproduced, modified, transmitted or
+-    transferred in any form or by any means, electronic or mechanical,
+-    without the express permission of the University. The permission of
+-    the University is not required if the said reproduction, modification,
+-    transmission or transference is done without financial return, the
+-    conditions of this Licence are imposed upon the receiver of the
+-    product, and all original and amended source code is included in any
+-    transmitted product. You may be held legally responsible for any
+-    copyright infringement that is caused or encouraged by your failure to
+-    abide by these terms and conditions.
+-
+-    You are not permitted under this Licence to use this Software
+-    commercially. Use for which any financial return is received shall be
+-    defined as commercial use, and includes (1) integration of all or part
+-    of the source code or the Software into a product for sale or license
+-    by or on behalf of Licensee to third parties or (2) use of the
+-    Software or any derivative of it for research with the final aim of
+-    developing software products for sale or license to a third party or
+-    (3) use of the Software or any derivative of it for research with the
+-    final aim of developing non-software products for sale or license to a
+-    third party, or (4) use of the Software to provide any service to an
+-    external organisation for which payment is received. If you are
+-    interested in using the Software commercially, please contact Oxford
+-    University Innovation ("OUI"), the technology transfer company of the
+-    University, to negotiate a licence. Contact details are:
+-    fsl@innovation.ox.ac.uk quoting Reference Project 9564, FSL.*/
```

Comment:
@casparvl It seems like this patch can be shortened significantly by not stripping out the copyright header?

Reply:
I know, but it's a lot of files... I've just taken the patched files supplied by the devs and create this patch with a ```diff -Nru```... Do you terribly mind the clutter? It's only for FSL-6.0.2, the devs have implemented in from FSL-6.0.3 onwards anyway...

Reply:
@boegel Can you please let me know if the copyright header is a dealbreaker for you? (see above comment) I think it is the last thing potentially blocking this PR...

Reply:
@casparvl A deal breaker is a bit strongly worded, but I do feel it's not a good idea to strip out copyright headers, especially for FSL which is installed via a build-in-installdir installation procedure.
Almost half of the patch consists of only stripping out copyright headers btw, for some files being patched there actually are no code changes at all.

I took the effort of fixing the patch file myself, see https://github.com/sara-nl/easybuild-easyconfigs/pull/5

Reply:
Wow, well done! Last time I tried to manually touch patch files it took me a lot of effort =) (or did you just touch the source files and recreate a diff?)

 Anyway, I merged the PR at our repo, so I think this one should get automatically updated :)